crates/cargo-bsp/src/cargo_communication/request_handle.rs

```diff
@@ -48,6 +49,7 @@ impl RequestHandle {
             root_path,
             cargo_handle,
             cancel_receiver,
+            &global_state.workspace,
```

Comment:
dlaczego od razu nie przekazać całego `global_state` po prostu? Wtedy byśmy nie musieli osobno przesyłać `root_path` i `workspace`

Reply:
W `RequestHandle` (gdzie tworzymy `RequestActor`) i tak musimy wyciągnąć `root_path` na potrzeby funkcji tworzących komendy. Więc nie wiem czy jest sens przekazywać potem cały `global_state`, żeby wyciągnąć to po raz drugi

Reply:
okay, jak uważasz